src/app/theming/styles/themes.scss

```diff
@@ -1,5 +1,6 @@
 // Import the theme utilities first
 @import '~igniteui-angular/lib/core/styles/themes/utilities';
+@import '~igniteui-angular/lib/core/styles/themes/schemas/_index';
```

Comment:
No need to import this as it's already imported via utilities.

Reply:
The redundant import is removed